### ISSUE TITLE
Disabling mobs in Configs causes BN to insert null entities in 1.18.2

### ISSUE THREAD

**Issue by Laifsyn** [paulevsGitch/BetterNether#538]
### What happened?

So far only tested when disabling mobs. It's unknown if something else happens when disabling something else

[faulty mobs.json config as .txt](https://github.com/paulevsGitch/BetterNether/files/8547575/mobs.txt)

### BetterNether

better-nether-6.1.0

### BCLib

bclib-1.4.3

### Minecraft

1.18.1

### Relevant log output

```shell
[latest.log](https://github.com/paulevsGitch/BetterNether/files/8547572/latest.log)
```

**Comment by quiqueck** [comment]
Hm, I think it is actually the chair that is cuasing this. Coud you try to enable it and confirm that the crash is gone.

**Comment by quiqueck** [comment]
Should be fixed in the next release for 1.18.2
